lenses_python/lenses.py

```diff
@@ -1,50 +1,133 @@
-from requests import *
-from json import *
-
-from lenses_python.SqlHandler import SqlHandler as SqlH
-from lenses_python.TopicHandler import TopicHandler as TopicH
-from lenses_python.ProcessorHandler import ProcessorHandler as PrcH
-from lenses_python.SchemasHandler import SchemasHandler as SchemaH
-from lenses_python.ConnectorHandler import ConnectorHandler as ConnH
-from lenses_python.WebSocketHandler import SubscribeHandler as SubSH
-from lenses_python.PublishHandler import PublishHandler as PuHandl
-from lenses_python.ACLHandler import ACLHandler
-from lenses_python.QuotaHandler import QuotaHandler
-from  lenses_python.KerberosTicket import KerberosTicket
+from requests import post, get
+from json import dumps
 
 class lenses:
 
     def __init__(self, url, username="", password="", kerberos_mode=0):
         """
-
         :param url:
         :param username:
         :param password:
         :param kerberos_mode: if it's equal to 1 we have kerberos mode, in default the value is 0 so no kerberos_mode
         """
+
         self.url = url
         self.username = username
         self.password = password
+        self.len_mods()
+
         # Check if user use kerberos mode
         if kerberos_mode == 0:
             # If not get the token with basic authentication
             self._Connect()
+            self.default_headers = {'Content-Type': 'application/json', 'Accept': 'application/json',
+                                    'x-kafka-lenses-token': self.token}
         else:
             # else take the token with kerberos connect way
             self._KerberosConnect()
 
+    # CHECK THE LENSES PYMODS
+    def len_mods(self):
+        _check="[\033[5;32m * \033[0;0m]"
+        print(_check + " Checking lenses pylib modules.")
+
+        try:
+            from lenses_python.SqlHandler import SqlHandler as SqlH
+            from lenses_python.TopicHandler import TopicHandler as TopicH
+            from lenses_python.ProcessorHandler import ProcessorHandler as PrcH
+            from lenses_python.SchemasHandler import SchemasHandler as SchemaH
+            from lenses_python.ConnectorHandler import ConnectorHandler as ConnH
+            from lenses_python.WebSocketHandler import SubscribeHandler as SubSH
+            from lenses_python.PublishHandler import PublishHandler as PuHandl
+            from lenses_python.ACLHandler import ACLHandler
+            from lenses_python.QuotaHandler import QuotaHandler
+            from lenses_python.KerberosTicket import KerberosTicket
+
+        except ImportError:
+            raise
+            exit(1)
+
+        print(_check + " No issue detected.\n")
+        del _check
+        #global SqlH, TopicH, PrcH, SchemaH, ConnH, SubSH, PuHandl, ACLHandler, QuotaHandler, KerberosTicket
+        del SqlH, TopicH, PrcH, SchemaH, ConnH, SubSH, PuHandl, ACLHandler, QuotaHandler, KerberosTicket
+
+    def kopts(self, action, *opts, **kopts):
```

Comment:
Yes indeed. However I thought I could move this to a separate file and improve it to check all options and give suggestions based on the minimal requirements for each call.

But, since most of the dictionaries will be removed (based on the suggestions above), this method will be removed too.